app/components/FilterMenu.tsx

```diff
@@ -49,25 +49,41 @@ export const FilterMenu = ({ onClose }: FilterMenuProps) => {
             borderBottomWidth={"1px"}
             borderBottomColor={"gray.400"}
           >
-            Filter by Geography
+            Filter by District
           </Heading>
         </Show>
-        <FormControl id="geographyType">
-          <FormLabel>Geography Type</FormLabel>
+        <FormControl id="districtType">
+          <FormLabel
+            onClick={() => {
+              onClose();
+            }}
+          >
+            District Type
+          </FormLabel>
           <Select
             placeholder="-Select-"
             variant="base"
             onChange={(e: FormEvent<HTMLSelectElement>) => {
-              setGeorgaphyType(e.currentTarget.value as GeographyType);
+              console.log(e.currentTarget.value);
+              if (e.currentTarget.value === "") {
+                setSearchParams({}, { replace: true });
+              } else {
+                setSearchParams(
+                  {
+                    districtType: e.currentTarget.value,
+                  },
+                  { replace: true },
+                );
+              }
             }}
-            value={geographyType}
+            value={districtType === null ? "" : districtType}
           >
             <option value={"cd"}>Community District</option>
             <option value={"ccd"}>City Council District</option>
           </Select>
         </FormControl>
         <HStack spacing={2} width={"full"}>
-          {geographyType !== "ccd" ? (
+          {districtType !== "ccd" ? (
```

Comment:
I'm familiar with the advice to use the full ternary operator in these cases to avoid the embarrassment of accidentally rendering "0". However, this is an actual boolean check. We can safely use the "&&" operator. This can help clean up the trailing "null" and keep things concise as we add more conditions.